lib/kamal/commands/builder/pack.rb

```diff
@@ -0,0 +1,27 @@
+class Kamal::Commands::Builder::Pack < Kamal::Commands::Builder::Base
+  def push
+    combine \
+      pack(:build,
+        config.repository,
+        "--platform", platform,
+        "--builder", pack_builder,
+        buildpacks,
+        "-t", config.absolute_image,
+        "-t", config.latest_image,
+        "--env", "BP_IMAGE_LABELS=service=#{config.service}",
+        *argumentize("--env", args),
```

Comment:
With docker, build args are passed as `--build-arg` and with Kamal you set them via:

```
  args:
    ENVIRONMENT: production
```

You'd still set "build args" with pack via the same `args` section but they ultimately get passed as `--env` to the pack command. Trying to reduce confusion of when to use env/arg if you're testing out your builds.